kratos/tests/cpp_tests/utilities/test_sensitivity_builder.cpp

```diff
@@ -38,7 +38,7 @@ class TestAdjoint : public TElement
     typedef Kratos::unique_ptr<TestAdjoint> UniquePointer;
 
 
-    static TestAdjoint::Pointer Create(std::size_t NewId, const PointerVector<Node<3>>& rNodes)
```

Comment:
As a curiosity, why the typename is needed now?

Reply:
It was coming from a error massage in Windows.